package.json

```diff
@@ -82,6 +82,7 @@
     "generate-alpharelease": "eslint-generate-prerelease alpha",
     "generate-betarelease": "eslint-generate-prerelease beta",
     "generate-rcrelease": "eslint-generate-prerelease rc",
-    "publish-release": "eslint-publish-release"
+    "publish-release": "eslint-publish-release",
+    "prepare": "npm run -s build"
```

Comment:
Please be sure to remove `prepare` before merging. We need different steps before publishing in order to generate `version.js`.

You should also be able to use `preinstall` instead of `prepare` to allow you to install from GitHub.

Reply:
```suggestion
```